04.ls/ls.rb

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+COL_MAX = 3
+PADDING = 2
+
+def main
+  path = ARGV[0] if ARGV[0]
```

Comment:
```suggestion
  path = ARGV[0]
```

ARGV[0]がnilならそのままnilが代入されるだけなので条件分岐は不要です